rdtools/soiling.py

```diff
@@ -701,3 +704,108 @@ def soiling_srr(energy_normalized_daily, insolation_daily, reps=1000,
         max_negative_step=max_negative_step)
 
     return sr, sr_ci, soiling_info
+
+
+def _count_month_days(start, end):
+    '''Return a dict of number of days between start and end (inclusive) in each month'''
+    days = pd.date_range(start, end)
+    months = [x.month for x in days]
+    out_dict = {}
+    for month in range(1, 13):
+        out_dict[month] = months.count(month)
+
+    return out_dict
```

Comment:
Is this a good use case for `collections.Counter` (which is a subclass of dict)?  For example `Counter(pd.date_range('2019-01-01', '2019-12-31').month)`

Reply:
Potentially, but I think it may make some of the logic later on (around line 781) more complicated because the current form relies on every month being in every dict, and zero counts being included.